Project/src/main/java/com/example/demo/service/Player.java

```diff
@@ -33,11 +33,21 @@ public void loadPlayersData() {
 
     public List<String[]> getPlayers() {
         Collection<String[]> values = playerMap.values();
-        return new ArrayList<>(values);
```

Comment:
Player is a model class, its bad practice to have business logic in here

the only thing Player should have is the fields of a player (id, age, ..) consructor + setters + getters (use lombok for that)

Please move the entire logic to a different class, like FileLoaderService

Reply:
Done